dm/master/bootstrap.go

```diff
@@ -70,6 +70,20 @@ func (s *Server) bootstrap(ctx context.Context) error {
 	return nil
 }
 
+func (s *Server) bootstrapBeforeSchedulerStart(ctx context.Context) error {
+	log.L().Info("bootstrap before scheduler start")
+	// no need for v1.0.x
+	if s.cfg.V1SourcesPath != "" {
+		return nil
+	}
+
+	err := upgrade.TryUpgradeBeforeSchedulerStart(ctx, s.etcdClient)
```

Comment:
```suggestion
	return upgrade.TryUpgradeBeforeSchedulerStart(ctx, s.etcdClient)
```

Reply:
seems our lint is broken 🤔

Reply:
Seems revive can only detect err declare in if
```golang
if err:=...;err!=nil{
    return err
}
return nil
```